custom_components/frigate/switch.py

```diff
@@ -41,7 +41,7 @@ async def async_setup_entry(
         entities.extend(
             [
                 FrigateSwitch(entry, frigate_config, camera, "detect"),
-                FrigateSwitch(entry, frigate_config, camera, "clips"),
+                FrigateSwitch(entry, frigate_config, camera, "recordings"),
```

Comment:
This will cause the old clips switch to be orphaned, as this effectively changes the unique_id. It would be  cleaner to add some logic here to explicitly delete the old clips switch from the entity registry before adding the new recordings switch.

Reply:
(Is the intention to retire the clips switch? Why? It's very useful!)

Reply:
@blakeblackshear hinted at this below, but `clips` as a concept will be going away. There will be `events` with associated snapshots/thumbnails, and `recordings` which are for the cameras. You will be able to configure retention settings for `recordings` which are based on either a static retention period, or set different retention for recording segments based on the existence of overlapping events. That configuration is identical to the `clips` configuration today.

The reasoning behind this is to simplify the architecture, as well as remove duplicate `mp4` files for different events that overlap the same recording files.  For example, a car pulling into the driveway and 3 people getting out of it can potentially have 4 different events associated with it which would cause 4 nearly identical clips to be created.  This will allow us to only retain the recordings that overlap those events, and save a lot of storage space in the process.

Reply:
Your assumptions are correct, and yes, we still need to clean up the old entities from the registry.